lib/node_modules/@stdlib/math/base/complex/cis/README.md

```diff
@@ -0,0 +1,95 @@
+# cis
+
+> Compute the [cis][cis] function of a complex number.
+
+<section class="intro"> 
+
+The [cis][cis] function is defined as
+
+<!-- <equation class="equation" label="eq:cis_function" align="center" raw="\operatorname{cis}(z) = e^{iz} = \cos(z) + i \sin(z)." alt="cis function"> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var cis = require( '@stdlib/math/base/complex/cis' );
+```
+
+#### cis( \[out,\] re, im )
+
+Evaluates the [cis][cis] function with a `complex` argument comprised of a _real_ component `re` and an _imaginary_ component `im`.
+
+``` javascript
+var v = cis( 0.0, 0.0 );
+// returns [ 1.0, 0.0 ]
+
+v = cis( 1.0, 0.0 );
+// returns [ ~0.540, ~0.841 ]
+```
+
+By default, the function returns real and imaginary components as a two-element `array`. To avoid unnecessary memory allocation, the function supports providing an output (destination) object.
+
+```javascript
+var out = new Float64Array( 2 );
+
+var v = cis( out, 1.0, 0.0 );
+// returns <Float64Array>[ ~0.540, ~0.841 ]
+
+var bool = ( v === out );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="examples">
+
+## Examples
+
+``` javascript
+var Complex128 = require( '@stdlib/types/complex/float64' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var round = require( '@stdlib/math/base/special/round' );
+var real = require( '@stdlib/types/complex/real' );
+var imag = require( '@stdlib/types/complex/imag' );
+var cis = require( './../lib' );
```

Comment:
This `require` statement should be the package name. This is not the case for the actual example file which can use a relative `require`.

Reply:
Good catch. Seems I just copy-pasted this. ✅